packages/config/config/devices/0x0190/adc-t_3000.json

```diff
@@ -0,0 +1,804 @@
+{
+	"manufacturer": "Building 36 Technologies",
+	"manufacturerId": "0x0190",
+	"label": "ADC-T 3000",
+	"description": "Alarm.com Smart Thermostat",
+	"devices": [
+		{
+			"productType": "0x0006",
+			"productId": "0x0001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "HVAC System Type",
+			"description": "Configures the type of heating system used.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Normal",
+					"value": 0
+				},
+				{
+					"label": "Heat Pump",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Number of Heat Stages",
+			"description": "Heat Stages 0-3 Default is 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2
+		},
+		{
+			"#": "3",
+			"label": "Number of Cool Stages",
+			"description": "Cool Stages 0-2 Default is 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2
+		},
+		{
+			"#": "4",
+			"label": "Heat Fuel Type",
+			"description": "Choose type of fuel. Reality - whether unit is boiler vs forced air.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Fossil Fuel",
+					"value": 0
+				},
+				{
+					"label": "Electric",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "5",
+			"label": "Calibration Temperature",
+			"unit": "°F",
+			"valueSize": 4,
+			"minValue": -10,
+			"maxValue": 10,
+			"defaultValue": 0
+		},
+		{
+			"#": "6",
+			"label": "Swing",
+			"unit": "0.5°F",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 1
+		},
+		{
+			"#": "7",
+			"label": "Overshoot",
+			"unit": "0.5°F",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0
```

Comment:
Something odd with all the parameters that are temperatures, here's how they show up:
![image](https://user-images.githubusercontent.com/2053039/151093089-c7d149a1-593b-4520-82b1-2006855a773f.png)

This is what the manual says:
> Temperature parameters take the form of 4-byte Z-Wave temperature commands where byte 1
is the (precision|scale|size) and bytes 2 and 3 are the 16-bit signed temperature value. Byte 4
is unused. The thermostat will format temperature reports with the same scale as the
thermostat display temperature units. If the temperature value is 0xFFFF (e.g. 0x2AFFFF00)
the parameter is disabled.

This is consistent with how z-wave encodes other values that have a bult-in precision and scale (from what I saw). Is there a way to specify these configuration parameters so they show up correctly in the UI?

Reply:
We currently don't have anything in place to say "this is a Z-Wave float". This is also the very first time I've seen it, so it doesn't seem to be common. You can split this parameter into 4 seperate partials for Precision/Scale/Size/Temperature to make it a bit easier to enter this for humans.